planet/System.c4g/LanguageDE.txt

```diff
@@ -783,7 +782,7 @@ IDS_MSG_CHATDISCLAIMER=Chat - Nutzungsbedingungen
 IDS_MSG_CLEARTHEMESSAGEBOARD=Alle Nachrichten l�schen.
 IDS_MSG_CLIENT=Client
 IDS_MSG_CMD_ABORT_NOCOUNTDOWN=Countdown l�uft nicht!
-IDS_MSG_CMD_HOSTONLY=Kein Host? Versagt!
+IDS_MSG_CMD_HOSTONLY=Nur der Host kann das machen.
```

Comment:
"kann das machen" sounds a bit too childish for me, maybe "kann diesen Befehl ausführen"? (Why ever this message needs to be changed)

Reply:
Changed 70b108f944a8692a99adb0246c551e55affb6dc5